PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+/// The Bar class
+public class Bar {
+    var yValues: [Double]
+    var label: String
+    var color: [UIColor]?
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    /// - parameter color: provide a color base on the class UIColor
+    public init(_ yValues: [Double], label: String, color: [UIColor]) {
+        self.yValues = yValues
+        self.label = label
+        self.color = color
+    }
+
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    public init(_ yValues: [Double], label: String) {
+        self.yValues = yValues
+        self.label = label
+    }
+}
+
+/// PredixBarChartView -- Bar Chart
+open class PredixBarChartView: BarChartView {
+    /// :nodoc:
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialize()
+    }
+
+    /// :nodoc:
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initialize()
+    }
+
+    /// Predix Mobile Bar chart initial values
+    fileprivate func initialize() {
+        legend.horizontalAlignment = .right
+        legend.verticalAlignment = .bottom
+        xAxis.labelPosition = .bottom
+        chartDescription?.enabled = false
+    }
+
+    /// Set message if there's no data provided
+    public func setNoDataText(message: String) {
```

Comment:
Avoid methods prefixed with "set" or "get"
In this case, why create this method, as it just sets an existing public property?
It would be better to expose this property with @IBInspectable so it can be set via Interface Builder.

Reply:
There's no set or get prefixed in the PredixBarChartView file. The properties are exposed with @IBInspectable from the PredixBarChartView+IBInspectable